NewSpring/ApollosProject
# (DEV-3703) Adds Dynamic Prayer Menu

## DESCRIPTION

<img width="447" alt="Screen Shot 2019-05-22 at 2 13 10 PM" src="https://user-images.githubusercontent.com/2659478/58198129-c0ec5680-7c9b-11e9-967e-a9a11e5c8f8c.png">


### What does this PR do, or why is it needed?

Makes the prayer menu tabs driven off content channels and user membership.

### How do I test this PR?

1. Check the `prayerMenuCategories` query. If not apart of a campus, should not return the campus category. Group membership isn't done yet but that's a different issue :)
2. Check to make sure the menu shows up with the custom images and descriptions from the Rock content channel.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
